go.mod

```diff
@@ -21,12 +19,114 @@ require (
 	k8s.io/component-helpers v0.23.3
 	k8s.io/klog/hack/tools v0.0.0-20210917071902-331d2323a192
 	k8s.io/klog/v2 v2.30.0
-	k8s.io/kube-aggregator v0.0.0
-	k8s.io/kube-openapi v0.0.0-20211115234752-e816edb12b65
 	k8s.io/kube-scheduler v0.23.3
 	k8s.io/kubernetes v1.23.3
 	k8s.io/utils v0.0.0-20211116205334-6203023598ed
-	sigs.k8s.io/yaml v1.2.0
+	sigs.k8s.io/controller-runtime v0.11.1
+	sigs.k8s.io/yaml v1.3.0
+)
+
+require (
```

Comment:
This new block is due to go 1.17?

Reply:
Right I'm not sure why these indirect requires are being added after running
```
go mod tidy
```

I took the first one `github.com/Azure/go-ansiterm` for example and did
```
$ go mod why -m github.com/Azure/go-ansiterm
# github.com/Azure/go-ansiterm
sigs.k8s.io/scheduler-plugins/cmd/scheduler
k8s.io/kubernetes/cmd/kube-scheduler/app
k8s.io/component-base/term
github.com/moby/term
github.com/moby/term/windows
github.com/Azure/go-ansiterm
```

Reply:
> Right I'm not sure why these indirect requires are being added after running

It seems to be a result of Go 1.17: https://stackoverflow.com/questions/69432877/why-there-are-two-require-blocks-in-go-mod-since-go-1-17

> Because in Go 1.17 the module graph has been changed to enable pruning and lazy loading. The second require block contains indirect dependencies.